src/NaNMath.jl

```diff
@@ -57,7 +57,7 @@ function sum(x::AbstractArray{T}) where T<:AbstractFloat
     end
 
     if isnan(result)
-        Base.warn_once("All elements of the array, passed to \"sum\" are NaN!")
+        @warn "All elements of the array, passed to \"sum\" are NaN!"
```

Comment:
Missing `using Compat`? This is failing on 0.6.

Reply:
Just fixed it